funnel/forms/project.py

```diff
@@ -82,11 +86,21 @@ class ProjectForm(forms.Form):
             forms.validators.Length(max=2000),
         ],
     )
+    description = forms.MarkdownField(
+        __("Project description"),
+        validators=[forms.validators.DataRequired()],
+        description=__("Landing page contents"),
+    )
+
+    def validate_location(self, field):
+        if re.findall(r'["”]+', field.data):
+            raise forms.ValidationError(__("Double quote not allowed in location name"))
```

Comment:
IMO, if using regex, always pre-compile using `re.compile` at module or class level. `re.search` will do as we don't care for _all_ instances. The regex only needs to be `["“”]` (without the `+`, but including closing quote for possible partial copy-paste)

Error message: "Quotes are not necessary in the location name"